src/snowflake/snowpark/session.py

```diff
@@ -113,6 +114,10 @@
 
 _session_management_lock = RLock()
 _active_sessions: Set["Session"] = set()
+_use_scoped_temp_objects = True
```

Comment:
let's put it in context.py, which is similar to the opt-out option of @sfc-gh-yixie's simplifier? Also, do we need prefix underscore?

Reply:
moving this into `context.py` would result in circular imports.. I don't have a good solution for this (I do not want to do import while in the run time..) since I want to keep this parameter private, I think it's fine to leave it in the session.py?

also for the underscore, this change should be transparent to users, however, we do want to have a flag on client side to turn it off in case something went wrong.

and ultimately, we can just remove this config.

Reply:
can you just reference `snowflake.snowpark.context._use` here? I'd prefer to keep these options in one place and consistent.

Reply:
based upon our OOB discussion, removed the parameter to context. thanks for the suggestion!